src/js/common/schemaform/SchemaForm.jsx

```diff
@@ -26,7 +27,7 @@ const fields = {
 
 const reviewFields = {
   ObjectField: ReviewObjectField,
-  ArrayField,
+  ArrayField: ReadOnlyArrayField,
```

Comment:
No, what we've been doing is pulling array fields out of the schema and displaying them separately. That's still a thing and we use the review version of ArrayField there. 

This change means that, if you don't do that (with the `keepArrayInPageOnReview` option), you get this component in view mode.